corehq/apps/userreports/models.py

```diff
@@ -94,6 +94,7 @@ class DataSourceConfiguration(UnicodeMixIn, CachedCouchDocumentMixin, Document):
     meta = SchemaProperty(DataSourceMeta)
     is_deactivated = BooleanProperty(default=False)
     last_modified = DateTimeProperty()
+    asynchronous = BooleanProperty(default=False)
```

Comment:
Is this something that would get set manually? Who would decide whether or not to set this?